packages/dart_frog_cli/lib/src/commands/dev/dev.dart

```diff
@@ -74,12 +74,18 @@ class DevCommand extends DartFrogCommand {
         _sigint = sigint ?? io.ProcessSignal.sigint,
         _startProcess = startProcess ?? io.Process.start,
         _generatorTarget = generatorTarget ?? _defaultGeneratorTarget {
-    argParser.addOption(
-      'port',
-      abbr: 'p',
-      defaultsTo: '8080',
-      help: 'Which port number the server should start on.',
-    );
+    argParser
+      ..addOption(
+        'port',
+        abbr: 'p',
+        defaultsTo: '8080',
+        help: 'Which port number the server should start on.',
+      )
+      ..addOption(
+        'dart-vm-port',
```

Comment:
```suggestion
        'service-port',
```

Reply:
Yeah, that is a good call, @easazade can you address this comment, I will merge it once this change is made :)

Reply:
makes sense